### DISCUSSION TITLE
Question about KafkaConnect rollingUpdate

### DISCUSSION THREAD

**Discussion by BadCandy** [strimzi/strimzi-kafka-operator#10135] [Q&A] [answered]
Hello,

I have a question about Kafka Connect rolling update.

When performing a rolling update on a Kafka Connect Cluster, I have confirmed that the pods are updated one at a time.
Is there a way to configure it so that more than one pod is updated simultaneously?
I looked through the documentation and CRD but couldn't find a method for this.
Is this a feature that is not supported?

There was [a similar question](https://github.com/orgs/strimzi/discussions/8208#discussion-4939288) in the past, but it doesn't seem to have been supported.

**Comment by scholzj** [comment]
It is currently always one pod at a time.

**Reply by BadCandy** [reply]
Thank you for reply. 
Do you have any plans to support this feature in the future?

**Reply by scholzj** [reply, marked as answer]
There are no plans for it currently. If you want, you can open an enhancement issue and describe there the use-case as well as how would you expect it to work. TBH, I'm not sure it is completely trivial and not sure how well it would work with Connect and how it schedules the connectors. So it would need to be clarified I guess.

**Reply by BadCandy** [reply]
Ok, Thank you!
